ui/src/onboarding/components/SigninForm.scss

```diff
@@ -1,8 +1,38 @@
-.cf-panel.signin-panel {
+.signin-page {
+  .cf-funnel-page--content {
+    display: flex;
+    justify-content: center;
+    height: 100%;
+  }
+
+  .cf-dapper-scrollbars--content {
+    display: flex !important;
```

Comment:
is  this `!important` necessary?

Reply:
Unfortunately yes, in clockface there's an `!important` present to override styles from a library, so the madness trickles down